export_minimal_environment.bat

```diff
@@ -38,65 +38,68 @@ if exist "%ENVIRONMENT_PATH%\build_environment.log" (
-%COPY% "%QT_DIR%\src\opengl\vc140.pdb"  "%QT_MINIMAL_DIR%\lib\QtOpenGL.pdb"
-%COPY% "%QT_DIR%\src\plugins\accessible\widgets\vc140.pdb"  "%QT_MINIMAL_DIR%\plugins\accessible\qtaccessiblewidgets.pdb"
-%COPY% "%QT_DIR%\src\plugins\iconengines\svgiconengine\vc140.pdb"  "%QT_MINIMAL_DIR%\plugins\iconengines\qsvgicon.pdb"
-%COPY% "%QT_DIR%\src\plugins\imageformats\ico\vc140.pdb"  "%QT_MINIMAL_DIR%\plugins\imageformats\qico.pdb"
-%COPY% "%QT_DIR%\src\plugins\imageformats\svg\vc140.pdb"  "%QT_MINIMAL_DIR%\plugins\imageformats\qsvg.pdb"
-%COPY% "%QT_DIR%\src\plugins\imageformats\tga\vc140.pdb"  "%QT_MINIMAL_DIR%\plugins\imageformats\qtga.pdb"
-%COPY% "%QT_DIR%\src\script\vc140.pdb"  "%QT_MINIMAL_DIR%\lib\QtScript.pdb"
-%COPY% "%QT_DIR%\src\scripttools\vc140.pdb"  "%QT_MINIMAL_DIR%\lib\QtScriptTools.pdb"
-%COPY% "%QT_DIR%\src\sql\vc140.pdb"  "%QT_MINIMAL_DIR%\lib\QtSql.pdb"
-%COPY% "%QT_DIR%\src\svg\vc140.pdb"  "%QT_MINIMAL_DIR%\lib\QtSvg.pdb"
-%COPY% "%QT_DIR%\src\testlib\vc140.pdb"  "%QT_MINIMAL_DIR%\lib\QtTest.pdb"
-%COPY% "%QT_DIR%\src\xml\vc140.pdb"  "%QT_MINIMAL_DIR%\lib\QtXml.pdb"
-
-REM only copy compiled qt_ translations.
-%XCOPY% "%QT_DIR%\translations\qt_*.qm" "%QT_MINIMAL_DIR%\translations"
-REM The above also copies qt_help_*, which we don't need.
-del "%QT_MINIMAL_DIR%\translations\qt_help_*"
+%COPY% "%QT4_DIR%\src\gui\vc140.pdb"  "%QT4_MINIMAL_DIR%\lib\QtGui.pdb"
+%COPY% "%QT4_DIR%\src\network\vc140.pdb"  "%QT4_MINIMAL_DIR%\lib\QtNetwork.pdb"
+%COPY% "%QT4_DIR%\src\opengl\vc140.pdb"  "%QT4_MINIMAL_DIR%\lib\QtOpenGL.pdb"
+%COPY% "%QT4_DIR%\src\plugins\accessible\widgets\vc140.pdb"  "%QT4_MINIMAL_DIR%\plugins\accessible\qtaccessiblewidgets.pdb"
+%COPY% "%QT4_DIR%\src\plugins\iconengines\svgiconengine\vc140.pdb"  "%QT4_MINIMAL_DIR%\plugins\iconengines\qsvgicon.pdb"
+%COPY% "%QT4_DIR%\src\plugins\imageformats\ico\vc140.pdb"  "%QT4_MINIMAL_DIR%\plugins\imageformats\qico.pdb"
+%COPY% "%QT4_DIR%\src\plugins\imageformats\svg\vc140.pdb"  "%QT4_MINIMAL_DIR%\plugins\imageformats\qsvg.pdb"
+%COPY% "%QT4_DIR%\src\plugins\imageformats\tga\vc140.pdb"  "%QT4_MINIMAL_DIR%\plugins\imageformats\qtga.pdb"
+%COPY% "%QT4_DIR%\src\script\vc140.pdb"  "%QT4_MINIMAL_DIR%\lib\QtScript.pdb"
+%COPY% "%QT4_DIR%\src\scripttools\vc140.pdb"  "%QT4_MINIMAL_DIR%\lib\QtScriptTools.pdb"
+%COPY% "%QT4_DIR%\src\sql\vc140.pdb"  "%QT4_MINIMAL_DIR%\lib\QtSql.pdb"
+%COPY% "%QT4_DIR%\src\svg\vc140.pdb"  "%QT4_MINIMAL_DIR%\lib\QtSvg.pdb"
+%COPY% "%QT4_DIR%\src\testlib\vc140.pdb"  "%QT4_MINIMAL_DIR%\lib\QtTest.pdb"
+%COPY% "%QT4_DIR%\src\xml\vc140.pdb"  "%QT4_MINIMAL_DIR%\lib\QtXml.pdb"
+
+REM only copy compiled QT4_ translations.
+%XCOPY% "%QT4_DIR%\translations\QT4_*.qm" "%QT4_MINIMAL_DIR%\translations"
+REM The above also copies QT4_help_*, which we don't need.
+del "%QT4_MINIMAL_DIR%\translations\QT4_help_*"
+
+set QT5_VERSION=5.10.1
+%XCOPY% "%ENVIRONMENT_PATH%\Qt-%QT5_VERSION%" "%MINIMAL_PATH%\Qt-%QT5_VERSION%\"
```

Comment:
Is this to say that because of Qt5's build granularity that we are creating a 'minimal' environment by excluding even building certain modules, so we don't have to filter them here?

Reply:
Yea, I think copying individual files out of the result of `make install` is possibly going to get us into trouble. I think we could still do it though, but I can't tell what's needed or not in Qt5. I think adding more `-skip foo`s to `configure.bat` is a safer way to save space.